### DISCUSSION TITLE
V3 Window in Window?

### DISCUSSION THREAD

**Discussion by nikolaigeorgie** [wailsapp/wails#3220] [Q&A]
Would v3 have the capability to wrap a window that loads a url (such as google.com) rendering the `frontend`  as a header? (so that i can recreate a search input and trigger setUrl('') to navigate to new pages)

**Comment by leaanthony** [comment]
There won't be multiple embedded webviews per window so you'll need a html based solution

**Reply by nikolaigeorgie** [reply]
Thank you for the response!
